deploy/_move_tickets_to_done.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
This is a WIP to be moved to a new ticket as it is unnecessary for success of the epic